code/game/turfs/turf.dm

```diff
@@ -544,7 +547,7 @@
 
 
 /turf/proc/cultify()
-	ChangeTurf(/turf/space)
+	ChangeTurf(get_base_turf(src.z))
```

Comment:
er, it'll cultify to that turf, which'll cultify to that turf...

it won't be an infinite loop but it'll go on for infinity